### PR TITLE
home page refacto

### PR DESCRIPTION
**Pull request by LuckyShuii**
Refacto + upgraded back (handling situation if no main article or if main article isn't a published article)

### CODE REVIEW

**Review by Richarddesnoues — APPROVED** [review]

File: assets/styles/home.scss [hunk 1]
```diff
@@ -0,0 +1,383 @@
+/* card article principale */
+a {
```
Comment (Benja-san) [inline assets/styles/home.scss:2]: global behavior ? shift it into global css file

File: assets/styles/home.scss [hunk 2]
```diff
@@ -0,0 +1,383 @@
...
+    box-shadow: -0.7em 0.5em 0.5em -0.4em #00cc99;
+
+}
+
+ .little_article:hover {
+    rotate:Y -15deg;
+    transition: ease-in-out 0.7s;
+}
+
+main {
```
Comment (Benja-san) [inline assets/styles/home.scss:230]: global behavior ?

File: assets/styles/home.scss
Comment (Benja-san) [inline assets/styles/home.scss:1]: shift all global rules in order to avoid rules duplication in css files

**Review by Benja-san — CHANGES_REQUESTED** [review]

File: assets/styles/home.scss (hunk 2 above)
Comment (LuckyShuii) [reply assets/styles/home.scss:226]: it aint global

**Review by Benja-san — APPROVED** [review]

**Pull request merged**